libpod/networking_linux.go

```diff
@@ -154,13 +154,25 @@ func (r *Runtime) createNetNS(ctr *Container) (n ns.NetNS, q []*cnitypes.Result,
 	return ctrNS, networkStatus, err
 }
 
-func checkSlirpFlags(path string) (bool, bool, bool, error) {
+type slirpFeatures struct {
+	HasDisableHostLoopback bool
+	HasMTU                 bool
+	HasEnableSandbox       bool
+	HasEnableSeccomp       bool
+}
+
+func checkSlirpFlags(path string) (*slirpFeatures, error) {
 	cmd := exec.Command(path, "--help")
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		return false, false, false, errors.Wrapf(err, "slirp4netns %q", out)
-	}
-	return strings.Contains(string(out), "--disable-host-loopback"), strings.Contains(string(out), "--mtu"), strings.Contains(string(out), "--enable-sandbox"), nil
+		return nil, errors.Wrapf(err, "slirp4netns %q", out)
+	}
+	return &slirpFeatures{
```

Comment:
I just wonder if a log.debug statement here would be helpful to let us know slirp had been automatically enabled?